pkg/alerts/lcid_expiration_test.go

```diff
@@ -227,9 +227,9 @@ func TestShouldSendAlertForIntake_False(t *testing.T) {
 		assert.False(t, shouldSendAlertForIntake(intakeWithoutExpirationDate, now))
 	})
 
-	t.Run("skips intakes with a status of NO GOVERNANCE", func(t *testing.T) {
+	t.Run("skips intakes with a decision state of SIDS Not Governance", func(t *testing.T) {
 		intakeWithNoGovernance := testhelpers.NewSystemIntake()
-		intakeWithNoGovernance.Status = models.SystemIntakeStatusNOGOVERNANCE
```

Comment:
and because of the above  - double check

Reply:
In reference to my [previous comment](https://github.com/CMSgov/easi-app/pull/2435/files#r1499638157), I think we can probably just get rid of this test (since we already have a test that handles skipping items with retire dates)